upstream/pimlico_http_json_rpc_client.go

```diff
@@ -0,0 +1,188 @@
+package upstream
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/erpc/erpc/common"
+)
+
+var pimlicoSupportedChains = map[uint64]struct{}{
```

Comment:
hmm we're constantly adding new chains, would it be possible to make it dynamic? (eg by calling eth_chainId on the pimlico url to see if it's enabled for that chain)

Reply:
Yeah for sure that's possible within SupportsNetwork func which is exactly for this purpose. Will update